tests/components/bayesian/test_binary_sensor.py

```diff
@@ -223,9 +335,9 @@ def test_multiple_observations(self):
         self.hass.block_till_done()
 
         state = self.hass.states.get("binary_sensor.test_binary")
-        assert [{"prob_true": 0.8, "prob_false": 0.4}] == state.attributes.get(
-            "observations"
-        )
+
+        assert state.attributes.get("observations")[0]["prob_given_true"] == 0.8
+        assert state.attributes.get("observations")[0]["prob_given_false"] == 0.4
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5279916)